forge-gui/res/cardsfolder/upcoming/wasteful_harvest.txt

```diff
@@ -0,0 +1,9 @@
+Name:Wasteful Harvest
+ManaCost:2 G
+Types:Instant
+A:SP$ Mill | NumCards$ 5 | RememberMilled$ True | SubAbility$ DBChangeZone | SpellDescription$ Mill five cards. You may put a permanent card from among the cards milled this way into your hand.
+SVar:DBChangeZone:DB$ ChangeZone | Origin$ Graveyard | Destination$ Hand | Hidden$ True | ChangeType$ Permanent.IsRemembered | ChangeNum$ 1 | SelectPrompt$ Choose a permanent card milled this way | SubAbility$ DBCleanup
```

Comment:
My gut says `SelectPrompt$ You may choose`...

It's awesome being obsessive compulsive 😬